rasa/core/agent.py

```diff
@@ -234,9 +258,8 @@ async def load_agent(
         if endpoints.nlu:
             http_interpreter = RasaNLUHttpInterpreter(endpoints.nlu)
 
-        anonymization_pipeline = plugin_manager().hook.create_anonymization_pipeline(
-            anonymization_rules=endpoints.anonymization_rules,
-            event_broker_config=endpoints.event_broker,
+        anonymization_pipeline = create_anonymization_pipeline(
+            endpoints.anonymization_rules, endpoints.event_broker
```

Comment:
I actually don't think you need to replace the hook call here, this is fine as it is.